docs/configuration.md

```diff
@@ -0,0 +1,203 @@
+        {id: 'first-meaningful-paint', weight: 2, group: 'perf-metric'},
+        {id: 'first-interactive', weight: 3, group: 'perf-metric'},
+        {id: 'consistently-interactive', weight: 5, group: 'perf-metric'},
+      ],
+    }
+  }
+}
+```
+
+#### Options
+| Name | Type | Description |
+| -- | -- | -- |
+| name | `string` | The display name of the category. |
+| description | `string` | The displayed description of the category. |
+| audits | `Object[]` | The audits to include in the category. |
+| audits[$i].id | `string` | The ID of the audit to include. |
+| audits[$i].weight | `number` | The weight of the audit in the scoring of the category. |
+| audits[$i].group | `string` (optional) | The ID of the [display group](#groups-objectundefined) of the audit. |
+
+### `groups: Object|undefined`
+
+The groups property controls how to visually group audits within a category. For example, this is what enables the grouped rendering of metrics and accessibility audits in the report.
+
+**Note: logic to display audit groups is required in the report renderer. Adding arbitrary groups without additional rendering logic may not perform as expected.**
+
+#### Example
+```js
+{
+  categories: {
+    performance: {
+      audits: [
+        {id: 'my-performance-metric', weight: 2, group: 'perf-metric'},
+      ],
+    }
+  },
+  groups: {
+    'perf-metric': {
+      title: 'Metrics',
+      description: 'These metrics encapsulate your app\'s performance across a number of dimensions.'
+    },
+  }
+}
+```
+
+## Config Extension
+
+The stock Lighthouse configurations can be extended if you only need to make small tweaks, such as adding an audit or skipping an audit, but wish to still run most of what Lighthouse offers. When adding the `extends: 'lighthouse:default'` property to your config, the default passes, audits, groups, and categories will be automatically included, allowing you modify settings or add additional audits to a pass. See [more examples below](#more-examples) to see different types of extension in action.
+
+**Config extension is the recommended way to run custom Lighthouse**. If there's a use case that extension doesn't currently solve, we'd love to [hear from you](https://github.com/GoogleChrome/lighthouse/issues/new)!
+
+## More Examples
+
+The best examples are the ones Lighthouse uses itself! There are several reference configuration files that are maintained as part of Lighthouse.
+
+* [lighthouse-core/config/default.js](https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/config/default.js)
+* [lighthouse-core/config/perf.json](https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/config/perf.json)
+* [lighthouse-core/config/plots-config.js](https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/config/plots-config.js)
+* [docs/recipes/custom-audit/custom-config.js](https://github.com/GoogleChrome/lighthouse/blob/master/docs/recipes/custom-audit/custom-config.js)
```

Comment:
fwiw this is another: https://github.com/paulirish/pwmetrics/blob/master/lib/lh-config.ts

but its nearly identical to the plots config.